src/themes/default/components/core/Breadcrumbs.vue

```diff
@@ -12,41 +12,9 @@
 </template>
 
 <script>
-import { localizedRoute, currentStoreView } from '@vue-storefront/core/lib/multistore'
-import i18n from '@vue-storefront/i18n'
+import { Breadcrumbs } from '@vue-storefront/core/modules/breadcrumbs/components/Breadcrumbs.ts'
 
 export default {
-  computed: {
```

Comment:
Why did you move this logic to the mixin? breadcrumb it's a theme thing, this is the best place for it, we even try to eliminate core-components as much as it's possible, is there any reason?

Reply:
From a theme developer point of view, it's ideal to have logic and basic functionality in the core component mixins. Then the theme components are primarily for the presentation layer. Custom themes can easily incorporate the functionality and can still override anything.

For me, the list of routes used by the breadcrumbs is functional. How they are displayed is presentational and therefore in the theme.  This seemed to be the standard approach, at least until v1.10

> we even try to eliminate core-components as much as it's possible

Is this a new policy from v1.11?

Honestly, it will require so much more work for every custom theme to be updated with new features and bug fixes. It's very complicated to merge / copy changes from one component (default theme) into another (custom theme). It also results in a lot more repeated code, copied into every custom theme.

But if it's actually the new policy, of course I can easily return computed data and props back to the theme component. No problem.

Reply:
@grimasod Ok I see your point. Basically we try to move mixins/core components to the theme (if it makes sense, for sure we don't create new ones) but it doesn't mean that we delete them at all - always we mark them as deprecated. 

The reason of that is mixins causes problems such as name collision or they are hard do debug. Moreover, you have almost everything in the vuex store, so you can use actions/getters to create eg. your own breadcrumb or whatever you like - this is what you should use in your themes, it's much easier to maintain. 

Ok I think for now we can keep it as it is, you don't have to move it anywhere.

Honestly, based on your comment I'm considering to keep using mixins but only really small ones including really basic/default things such as that breadcrumb (it will be even same in most of themes). If that make theme developers lives easier - why not, but for sure we'll be continuing deprecating these complex ones that for sure won't be used by potential developer. (@patzick what do you think?)